.github/workflows/test.yml

```diff
@@ -273,11 +273,23 @@ jobs:
           name: Documentation HTML
           path: install/ddsrouter_docs/docs/ddsrouter_docs/sphinx/html/
 
+      - name: Concatenate spelling errors
+        run: |
+          SPELLING_ERRORS_DIR=./build/ddsrouter_docs/docs/test/spelling
+          if [ -d "$SPELLING_ERRORS_DIR" ]; then
+            find ./build/ddsrouter_docs/docs/test/spelling \
+              -type f \
+              -name '*.spelling' \
+              -exec cat {} + > ./build/ddsrouter_docs/test/spelling/spelling_errors.txt
+          else
+            echo "No spelling errors found."
+          fi
+
       - name: Upload spelling errors
         uses: actions/upload-artifact@v1
         with:
           name: spelling_errors.txt
-          path: build/ddsrouter_docs/docs/test/spelling/output.txt
+          path: build/ddsrouter_docs/test/spelling/spelling_errors.txt
```

Comment:
This fails if the file does not exist (and has a tag so job does not fail). Should we change it?